service/src/test/java/bio/terra/cbas/controllers/TestControllerUtils.java

```diff
@@ -0,0 +1,89 @@
+package bio.terra.cbas.controllers;
+
+import static bio.terra.cbas.util.BearerTokenFilter.ATTRIBUTE_NAME_TOKEN;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import bio.terra.cbas.dependencies.sam.SamService;
+import bio.terra.common.sam.exception.SamUnauthorizedException;
+import java.util.Optional;
+import org.broadinstitute.dsde.workbench.client.sam.model.UserStatusInfo;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.mock.web.MockHttpServletRequest;
+import org.springframework.web.context.request.RequestContextHolder;
+import org.springframework.web.context.request.ServletRequestAttributes;
+
+class TestControllerUtils {
+  private ControllerUtils utils;
+
+  private final String tokenValue = "foo-token";
+  private final String expiredTokenValue = "expired-token";
+  private final UserStatusInfo mockUser =
+      new UserStatusInfo()
+          .userEmail("realuser@gmail.com")
+          .userSubjectId("user-id-foo")
+          .enabled(true);
+
+  @BeforeEach
+  void init() {
+    SamService samService = mock(SamService.class);
+    when(samService.getUserStatusInfo(tokenValue)).thenReturn(mockUser);
+    when(samService.getUserStatusInfo(expiredTokenValue))
+        .thenThrow(new SamUnauthorizedException("Unauthorized :("));
```

Comment:
Was going to comment negatively about the `:(` but then I saw that this is in a test, and instead: 💯 🥇